src/components/frame/LandingPage/ColonyCards/CreateNewColonyCard.tsx

```diff
@@ -0,0 +1,75 @@
+import { Plus } from '@phosphor-icons/react';
+import React, { useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Button from '~v5/shared/Button/Button.tsx';
+
+import BaseColonyCard from './BaseColonyCard/BaseColonyCard.tsx';
+
+const displayName = 'frame.LandingPage.ColonyCards';
+
+export interface ColonyCreateCardProps {
+  invitationsRemaining: number;
+  onCreate: () => void;
+}
+
+const MSG = defineMessages({
+  remaining: {
+    id: `${displayName}.remaining`,
+    defaultMessage: `{remaining} INVITATIONS REMANING`,
+  },
+  createColony: {
+    id: `${displayName}.createColony`,
+    defaultMessage: `Create a new colony`,
+  },
+  createButton: {
+    id: `${displayName}.createButton`,
+    defaultMessage: `Create`,
+  },
+});
+
+export const CreateNewColonyCard = ({
+  invitationsRemaining,
+  onCreate,
+}: ColonyCreateCardProps) => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  return (
+    <BaseColonyCard
+      onClick={() => onCreate()}
+      onMouseOver={() => setIsHovered(true)}
+      onMouseLeave={() => setIsHovered(false)}
+      isClickable
+      avatarPlaceholder={
+        <div className="flex h-8 w-8 items-center justify-center rounded-full bg-gray-100">
+          <Plus size={18} className="text-gray-900 " />
+        </div>
+      }
+    >
+      <div className="flex flex-1 flex-col gap-1">
+        <div className="flex">
+          <span className="rounded bg-blue-100 px-[.1875rem] py-[.1563rem] text-2xs font-extrabold text-blue-400">
```

Comment:
Font weight should be 700: `font-bold`

<img width="1141" alt="Screenshot 2024-10-26 at 00 15 57" src="https://github.com/user-attachments/assets/225a7afb-9650-402f-96b2-f5b19afc8a87">